.gitignore

```diff
@@ -6,4 +6,5 @@ settings_local.py
 .idea
 .tox
 pdc.egg-info/
+.vscode
```

Comment:
So I'm not the only one who started using vscode for Python :) ...thought the Vim plugin is still quite annoying.